docs/setup/discovery-server.md

```diff
@@ -0,0 +1,46 @@
+# Fast DDS Discovery Server
+
+This page gives instructions on how to use a Fast DDS discovery server with the Create® 3 robot as a client.
+For more information on the Fast DDS discovery server, please refer to the [eProsima documentation](https://fast-dds.docs.eprosima.com/en/latest/fastdds/ros2/discovery_server/ros2_discovery_server.html).
+
+*We’ll use the term "computer" to refer to any SBC (single board computer), virtual machine, personal computer, etc.*
+
+## Configure Server
+
+1. Start the discovery server on the computer that you wish to use as your server by replacing <SERVER_IP> with the server’s IP address.
```

Comment:
> by replacing <SERVER_IP> with the server’s IP address.
this sentence seems a little bit redundant and not very clear.

is the "server's IP" the computer IP ? if yes, I would change this into "the computer's IP address"

Reply:
Yes, fair point. In this case, the computer in use should be the server. I will update